_schema.graphql

```diff
@@ -4269,29 +4269,6 @@ type Mutation {
     input: UpdateConversationMutationInput!
```

Comment:
the order in which these are defined has changed, so the text ordering has changed, should be the same values